packages/integration-tests/sfnStep.js

```diff
@@ -54,12 +54,12 @@ class SfnStep {
     if (scheduleEvent.type !== this.startFailedEvent) {
       startEvent = this.getStartEvent(executionHistory, scheduleEvent, this);
 
-      if (startEvent !== null && startEvent.type !== this.startFailedEvent) {
+      if (startEvent && startEvent.type !== this.startFailedEvent) {
```

Comment:
So we're saying here if there is no start event, return null for the whole thing? Wouldn't we want the schedule event if that exists?

Reply:
Returning scheduleEvent with `null/undefined` startEvent and completeEvent doesn't help.

The null value (returned from getStepExecutionInstance ) gets filtered out here:
https://github.com/nasa/cumulus/blob/56d922f9a15286c99dfdbdc5b85acb964d2b13d6/packages/integration-tests/sfnStep.js#L92

Reply:
Ok, just checking we wouldn't need the schedule event.